packages/shared/src/request/comment.ts

```diff
@@ -0,0 +1,28 @@
+import { useMutation } from 'react-query';
+import { DELETE_COMMENT_MUTATION } from '../graphql/comments';
+import { EmptyResponse } from '../graphql/emptyResponse';
+import { Post } from '../graphql/posts';
+import { usePostComment } from '../hooks/usePostComment';
+import { useRequestProtocol } from '../hooks/useRequestProtocol';
+import { apiUrl } from '../lib/config';
+
+export const deleteComment = ({
+  commentId,
+  parentId,
+  post,
+}: {
+  commentId: string;
+  parentId: string | null;
+  post: Post;
+}): Promise<EmptyResponse> => {
+  const { deleteCommentCache } = usePostComment(post);
+  const { requestMethod } = useRequestProtocol();
+  const { mutateAsync: deleteCommentRequest } = useMutation<EmptyResponse>(
```

Comment:
Perhaps we can move this to the `graphQL` `comments` file like you did previously to have everything uniform?